completions/mn

```diff
@@ -0,0 +1,65 @@
+# bash completion for mn                              -*- shell-script -*-
+
+_mn()
+{
+    local cur prev words cword
+    _init_completion || return
+
+    case $prev in
+        --switch)
+             COMPREPLY=($( compgen -W 'default ivs lxbr ovs ovsbr ovsk user
+                 ' -- "$cur" ) )
+             return 0
+             ;;
+        --host)
+             COMPREPLY=($( compgen -W 'block random' -- "$cur" ) )
+             return 0
+             ;;
+        --controller)
+             COMPREPLY=($( compgen -W 'default none nox ovsc ref remote
+                 ryu' -- "$cur" ) )
+             return 0
+             ;;
+        --link)
+             COMPREPLY=($( compgen -W 'default ovs tc' -- "$cur" ) )
+             return 0
+             ;;
+        --topo)
+             COMPREPLY=($( compgen -W 'linear minimal reversed single torus
+                 tree ' -- "$cur" ) )
+             return 0
+             ;;
+        --test)
+             COMPREPLY=( $( compgen -W 'cli build pingall pingpair iperf
+                 all iperfudp none' -- "$cur" ) )
+             return 0
+             ;;
+        -v|--verbosity)
+             COMPREPLY=($( compgen -W 'info warning critical error debug
+                 output' -- "$cur" ) )
+             return 0
+             ;;
+        --placement)
+             COMPREPLY=($( compgen -W 'block random' -- "$cur" ) )
+             return 0
+             ;;
+        --custom)
+             _filedir
+             return 0
+             ;;
+         -h|--help|-c|--clean|--version|--listenport)
+             return 0
+             ;;
+    esac
+
+    COMPREPLY=( $( compgen -W '-h -c -x -i -v
```

Comment:
@scop do you mean you don't want the short options be shown in the option list when a user is using the [tab] button?